### ISSUE TITLE
Snag list for the Entry Information Page

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#313]
- [x] Image link is undented (needs to be not indented
 - [x] Image link needs an image to clue the user
 - [x] Make postem link bold (like the scan link)
 - [x]  the styling that is applied to the icons on :hover needs to be extended to :focus so that a sighted keyboard-only user will see the same information that becomes visible on :hover.
 - [x] When the extra info appears after selecting the Postem icon or the scan icon, it is uncomfortably close to the navigation immediately above. I suggest you add the push--top class to its containing div.

**Comment by AlOneill** [comment]
In the details table, why is 'Page' rendered in a different (serif) font from all the other row headings?

Row headings could do with `scope="row"`.

The meaning of the images for 'View postem', 'View scan' and 'Save entry' is currently invisible to sighted keyboard-only users. I have not tested what happens for screen-reader users (cannot tell from markup; no alt text apparent).

Think this is covered elsewhere: casing of forenames should be Title case (not all caps).

**Comment by AlOneill** [comment]
I see that casing of Forenames has been changed, but it is not quite right: for example,
Edith daisy DENT should be Edith Daisy DENT.

**Comment by AlOneill** [comment]
In addition to my third comment, 2 posts above, the information that you can 'Click to view other entries on this page' is also not visible to sighted keyboard-only users because it is buried in a `title` attribute.

**Comment by Vino-S** [comment]
Fixed name casing issue
_In the details table, why is 'Page' rendered in a different (serif) font from all the other row headings?_
I am not sure why I added serif styling. Removed it
Added  scope="row" to Row Heading
Added accessibility text to Page number link.

**Comment by richpomfret** [comment]
@AlOneill to test out the latest changes.

**Comment by AlOneill** [comment]
Name casing now good except for examples that appear in the Results listing in all-caps, like this: ROBERTA ELIZABETH BRANDIS — these might be addressed as part of the larger review of name-casing #387

'Page' font is fixed. 'scope` attributes good. Thank you.

Cannot see accessibility text on page number link — but it would not help a keyboard-only user **who can see**. And it would annoy a screen-reader user to hear the contents of the `title` attribute and the accessibility text.

We have to make that text visible (would a `.tooltip` work? I cannot remember!) and delete the `title` attribute. If a tooltip will not work here then it would have to be plain text. Sorry. Unless anyone has a better idea??

Similarly, the text for the icons for 'Postems', 'View scan' and 'Save entry' is still hidden from a sighted keyboard-only user. If `:focus` can share the same properties as `:hover`, that would solve things for the icons.

**Comment by AlOneill** [comment]
Have just noticed a missing space, an unnecessary word and dodgy link-text in "If the scan does not contain the entry you may be able to find the scan yourself by clicking [here(link opens in new tab)]"

Suggest: "If the scan does not contain the entry you might be able to [find the scan yourself (opens in new tab)]". 

The words in square brackets are the the link-text, current and suggested.

**Comment by PatReynolds** [comment]
Thanks @AlOneill The proposed text is great.

**Comment by richpomfret** [comment]
Moved onto test3 for testing.

**Comment by richpomfret** [comment]
@AlOneill please take a look and give it a test if you get a chance.

**Comment by AlOneill** [comment]
@Vino-S Most of the changes are good, thank you. However, a _sighted_ keyboard-only user will not see the meaning of the three icons (two icons for those not logged in, which is what we have to fix for a researcher) in "More information". I do not have a good suggestion for how to fix these, but will think on.

When the extra info appears after selecting the Postem icon or the scan icon, it is uncomfortably close to the navigation immediately above. I suggest you add the `push--top` class to its containing `div`.

**Comment by AlOneill** [comment]
@Vino-S My thoughts have cleared: the styling that is applied to the icons on `:hover` needs to be extended to `:focus` so that a sighted keyboard-only user will see the same information that becomes visible on `:hover`.

**Comment by PatReynolds** [comment]
@Vino-S could you take a look at this? Thanks, Pat

**Comment by Vino-S** [comment]
Ready for testing in test3

**Comment by PatReynolds** [comment]
@PatReynolds and @DeniseColbert to test

**Comment by AlOneill** [comment]
@Vino-S There is still no information visible on tabbing into the Postem icon or the Scan icon — a sighted keyboard-only user will not see the contents of `span.accessibility` (although it will help a screen-reader user). If the `:focus` styling is adjusted (somehow!) to be the same as the `:hover` styling, that _should_ fix it.

_But_, looking again, the `:hover` and `:focus` states are set on a `div` which does not natively accept focus — it is the `a` element that is accepting the focus. Is it an issue with the JS? Any solution will have to be confirmed to also work in Safari which is finicky about such things. Good luck!

The spacing issue when a More Information icon is selected is fixed — thank you.

**Comment by Vino-S** [comment]
Ready for testing in test2

**Comment by AlOneill** [comment]
@Vino-S On test2, there is now **no text visible** on hover or focus for the postem and scan icons (Firefox, Safari not tested).

**Comment by Vino-S** [comment]
Thanks for testing @AlOneill ,
Updated test2 with hover and focus text

**Comment by AlOneill** [comment]
@Vino-S Text now visible on focus and hover (Safari and Firefox) — thank you.
